Dockerfile

```diff
@@ -36,6 +36,8 @@ RUN set -xe \
       libxslt \
       libmemcached-libs \
       libzip \
+    && apk add --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted \
```

Comment:
Glad to see there is an alpine package for memprof. :)

Reply:
Thanks to @sevein for pointing out this apk package